CHANGELOG.md

```diff
@@ -1,6 +1,189 @@
 Moto Changelog
 ===================
 
+1.3.14
+-----
+
+    General Changes:
+        * Support for Python 3.8
+        * Linting: Black is now enforced.
+
+    New Services:
+        * Athena
+        * Config
+        * DataSync
+        * Step Functions
+
+    New methods:
+        * Athena:
+            * create_work_group()
+            * list_work_groups()
+        * API Gateway:
+            * delete_stage()
+            * update_api_key()
+        * CloudWatch Logs
+            * list_tags_log_group()
+            * tag_log_group()
+            * untag_log_group()
+        * Config
+            * batch_get_resource_config()
+            * delete_aggregation_authorization()
+            * delete_configuration_aggregator()
+            * describe_aggregation_authorizations()
+            * describe_configuration_aggregators()
+            * describe_identity_pool()
```

Comment:
This is not a config method.

Reply:
Good spot!
I've raised https://github.com/spulec/moto/pull/2560 to remedy this